resources/services/iam/groups_access_advisor.go

```diff
@@ -0,0 +1,178 @@
+package iam
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/iam"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/diag"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+//go:generate cq-gen --resource groups_access_advisor --config gen.hcl --output .
+func GroupsAccessAdvisors() *schema.Table {
+	return &schema.Table{
+		Name:          "aws_iam_groups_access_advisor",
+		Resolver:      fetchIamGroupsAccessAdvisors,
+		Multiplex:     client.AccountMultiplex,
+		IgnoreError:   client.IgnoreAccessDeniedServiceDisabled,
+		DeleteFilter:  client.DeleteAccountFilter,
+		IgnoreInTests: true,
+		Options:       schema.TableCreationOptions{PrimaryKeys: []string{"account_id", "service_namespace"}},
```

Comment:
If we are going to use the same table for multiple entities then the PKs need something to differentiate between groups/roles and polices

Reply:
But if all of the data is the same except for a single column at the root table don't you think it would be simpler from a user perspective when querying?

Reply:
I can see tradeoffs on both sides of this. So I want to get others opinions

@roneli @zagronitay @yevgenypats  

We are working on implementing [4 resources](https://github.com/cloudquery/cq-provider-aws/issues/730) that have exactly the same fields. Do you think it is better to put the data into a single table and have a field in the root table to identify which resource type it is or have 4 different tables and infer the resource type by the table name?